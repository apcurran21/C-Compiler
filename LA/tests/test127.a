void main ( ){

  :entry
  int64[][] a
  int64 i
  int64 j

  a <- new Array(10, 10)
  i <- 1
  j <- -1
  a[i][j] <- 21

  return 

}
