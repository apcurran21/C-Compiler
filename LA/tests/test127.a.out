ERROR: instruction at line 11 attempted to use the negative index -1 in the dimension 1
